project/cmake/scripts/ios/ArchSetup.cmake

```diff
@@ -13,6 +13,7 @@ if(WITH_ARCH)
   set(ARCH ${WITH_ARCH})
 else()
   if(CPU STREQUAL armv7 OR CPU STREQUAL arm64)
+    set(CMAKE_OSX_ARCHITECTURES armv7)
```

Comment:
set CMAKE_OSX_ARCHITECTURES CPU? - we want to be able to build arm64 aswell ... also where does the CPU come from? i thought cmake variables must be references by ${CPU} then?